dev/reference/configuration/tikv-server/configuration-file.md

```diff
@@ -324,6 +324,16 @@ Configuration items related to Raftstore
 + Default value: `3s`
 + Minimum value: `0`
 
+### `raftstore.hibernate-regions` (**Experimental**)
+
++ Enables or disables Hibernate Region. When this option is enabled, a Region idle for a long time is automatically set as hibernated. This reduces the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle Regions. You can use `raftstore.peer-stale-state-check-interval` to modify the the heartbeat interval between the leader and followers of hibernated Regions.
```

Comment:
So why "the" before "followers" is used in the second sentence, but removed in the third one?

Reply:
updated. Let's merge this.